pkgs/tools/security/seclists/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+}:
+
+stdenv.mkDerivation rec {
+  pname = "seclists";
+  version = "2023.3";
+
+  src = fetchFromGitLab {
+    group = "kalilinux";
+    owner = "packages";
+    repo = "seclists";
+    rev = "upstream/${version}";
+    hash = "sha256-mJgCzp8iKzSWf4Tud5xDpnuY4aNJmnEo/hTcuGTaOWM=";
+  };
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp -r * $out/bin
```

Comment:
The output should not go to `bin` from my point of view. Some tools are already using `wordlists`. E. g., 

```nix
  outputs = [
    "wordlists"
  ];

  postInstall = ''
    mkdir -p $wordlists
    cp -R * $wordlists
  '';
```

It would probably be required to adjust the directory names as directories with names that start with an upper case letter are a bit uncommon and will make it hard to navigate as user wouldn't except it. At the other hand if one comes from the repo then the names match.